pkg/services/pluginsintegration/clientmiddleware/logger_middleware.go

```diff
@@ -51,7 +51,7 @@ func (m *LoggerMiddleware) logRequest(ctx context.Context, fn func(ctx context.C
 		logParams = append(logParams, "error", err)
 	}
 	if m.features.IsEnabled(featuremgmt.FlagPluginsInstrumentationStatusSource) {
-		logParams = append(logParams, "status_source", pluginrequestmeta.StatusSourceFromContext(ctx))
+		logParams = append(logParams, "statusSource", pluginrequestmeta.StatusSourceFromContext(ctx))
```

Comment:
Not related to this PR, but I realized that I used snake case for `status_source` for the other log line, so I'm converting it to camel case for consistency